usr/lib/jargonaut/jargonaut.py

```diff
@@ -420,13 +420,13 @@ def render_html(self):
     @idle
     def print_message(self, nick, text):
```

Comment:
Can you rename 'text' to 'message' here (and within the method also)

Reply:
Actually why don't you just reverse this PR - change text to message, and change the Message() on line 440 to something like _message. This will have far less changes.

Reply:
Thanks! I've updated the PR.